repos/fdbt-site/src/constants/index.ts

```diff
@@ -82,7 +82,15 @@ export const INTERNAL_NOC = 'IWBusCo';
 
 export const CREATED_FILES_NUM_PER_PAGE = 10;
 
-export const validFareTypes = ['single', 'period', 'return', 'flatFare', 'multiOperator', 'schoolService'];
+export const validFareTypes = [
+    'single',
+    'period',
+    'return',
+    'flatFare',
+    'multiOperator',
+    'schoolService',
+    'cappedProduct',
```

Comment:
I don't think we should be adding this here - capped products can be single or return or period etc.